architecture/service_catalog/service_catalog_cli.adoc

```diff
@@ -0,0 +1,439 @@
+----
+$  svcat describe broker ansible-service-broker
+  Name:     ansible-service-broker
+  URL:      https://openshift-automation-service-broker.openshift-automation-service-broker.svc:1338/openshift-automation-service-broker/
+  Status:   Ready - Successfully fetched catalog entries from broker @ 2018-06-07 00:32:59 +0000 UTC
+----
+
+[[service-catalog-cli-view-service-classes-and-plans]]
+=== View service classes and service plans
+When you create a `ClusterServiceBroker` resource, the service catalog
+controller queries the broker server to find all services it offers and creates
+a service class (`ClusterServiceClass`) for each of those services.
+Additionally, it also creates service plans (`ClusterServicePlan`) for each of
+the broker’s services.
+
+[[service-catalog-cli-view-serviceclass]]
+==== View service classes
+To view the available ClusterServiceClass resources:
+[source, bash]
+----
+$ svcat get classes
+                      NAME                                                                         DESCRIPTION                                                                     UUID
++-----------------------------------------------+----------------------------------------------------------------------------------------------------------------+--------------------------------------+
+  rh-mediawiki-apb                                Mediawiki apb implementation                                                                                     03b69500305d9859bb9440d9f9023784
+
+...
+
+  rh-mariadb-apb                                  Mariadb apb implementation                                                                                       2c259ddd8059b9bc65081e07bf20058f
+  rh-mysql-apb                                    Software Collections MySQL APB                                                                                   73ead67495322cc462794387fa9884f5
+  rh-postgresql-apb                               SCL PostgreSQL apb implementation                                                                                d5915e05b253df421efe6e41fb6a66ba
+
+----
+
+To view details of a service class:
+[source, bash]
+----
+$ svcat describe class rh-postgresql-apb
+  Name:          rh-postgresql-apb
+  Description:   SCL PostgreSQL apb implementation
+  UUID:          d5915e05b253df421efe6e41fb6a66ba
+  Status:        Active
+  Tags:          database, postgresql
+  Broker:        ansible-service-broker
+
+Plans:
+  NAME            DESCRIPTION
++------+--------------------------------+
+  prod   A single DB server with
+         persistent storage
+  dev    A single DB server with no
+         storage
+----
+
+[[service-catalog-cli-view-serviceplans]]
+==== View service plans
+To view the ClusterServicePlan resources available in the cluster, run the
+command:
+[source, bash]
+----
+$ svcat get plans
+   NAME                         CLASS                                DESCRIPTION                             UUID
```

Comment:
This part also need to update. The new version svcat doesn't include UUID in get class|plan output.
```
$ svcat get plans
   NAME           CLASS                  DESCRIPTION            
+---------+-------------------+--------------------------------+
  default   rh-mediawiki-apb    An APB that deploys MediaWiki   
  
  ...
  
  prod      rh-mariadb-apb      This plan deploys a single      
                                MariaDB instance with 10 GiB    
                                of persistent storage           
  dev       rh-mariadb-apb      This plan deploys a single      
                                MariaDB instance with           
                                ephemeral storage               
  prod      rh-mysql-apb        A MySQL server with persistent  
                                storage                         
  dev       rh-mysql-apb        A MySQL server with ephemeral   
                                storage                         
  prod      rh-postgresql-apb   A single DB server with         
                                persistent storage              
  dev       rh-postgresql-apb   A single DB server with no      
                                storage   
```